rust/src/lib.rs

```diff
@@ -250,21 +247,18 @@ pub extern "C" fn zcashlc_init_accounts_table(
         init_accounts_table(&db_data, &ufvks)
             .map(|_| {
                 // Return the Sapling ExtendedSpendingKeys for the created accounts.
-                let mut v: Vec<_> = usks
+                let v: Vec<_> = usks
                     .iter()
-                    .map(|(_, usk)| {
+                    .map(|(account, usk)| {
                         let encoded = encode_extended_spending_key(
                             network.hrp_sapling_extended_spending_key(),
                             usk.sapling(),
                         );
-                        CString::new(encoded).unwrap().into_raw()
+                        FFIEncodedKey::new(*account, encoded)
```

Comment:
This change makes me even more certain that this API should be returning the UFVKs instead of Sapling ExtFVKs (logically matching the _intent_ of returning the Sapling ExtFVKs previously). Non-blocking, this can be done in a separate PR.